ls1intum/Artemis
# Fixed navigation from tutor course dashboard

### Checklist

### Motivation and Context


Fixes #916 

### Description


### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Go to Tutor Course Dashboard
4. Go to Tutor Exercise Dashboard
5. Click back button twice and you should be on the Course Administration page again.
### Screenshots
